docs/readmes/proposals/p022_enodebd_enhancements.md

```diff
@@ -0,0 +1,177 @@
+---
+id: p022_enodebd_enhancements
+title: eNodeBd Enhancements for Cell PnP Support
+hide_title: true
+---
+
+Discussion at https://github.com/magma/magma/issues/10412
+
+# Proposal: eNodeBd Enhancements for Cell PnP Support
+
+Author(s): [@arsenii-oganov]
+
+Last updated: 11/17/2021
+
+### **1 Objectives**
+
+The objective of this work is to add services to address the logistics of scale radio deployments using Magma. **Specifically, plug-n-play onboarding and firmware update via TR-069**. 
+
+Plug-n-play onboarding enables auto-discovery and configuration of a new radio connected to a Magma network. This will enable less sophisticated users to add radios to a Magma managed network, as well making it quicker and easier for private cellular deployment users to setup and/or scale their networks. 
+
+Firmware update via TR-069 will allow the enodebd ACS to update the firmware on radios connected to the Magma network. Automated and Remote firmware updates are an important requirement for any scale deployment.  
+
+Software built to accomplish this will be open source under BSD-3-Clause license and will be committed to the Magma software repository under the governance of the Linux foundation, such that it can be effectively maintained in the future releases. The project will also enable endpoints for more advanced onboarding logic to allow for vendor extensions.
+
+To demonstrate the developed framework, full support for onboarding, configuration, and firmware update will be demonstrated for specific radio models including:
+- Baicells Nova 436Q
+- Baicells Nova 430
+- Baicells Neutrino 430
+- Sercomm Englewood 
+
+### **2 Background**
+
+**2.1: Terminology**
+
+1. **TR-069** refers to an application layer technical specification of the CPE WAN Management Protocol (CWMP) both specified by the Broadband Forum. This application is used for the remote management of configuration, firmware, and metrics in network equipment
+
+2. **ACS** refers to the Automatic Configuration Server which is the TR-069 server in a server client model.
+
```

Comment:
We should call out explicitly here the terminology and differences between magma `Managed` and `Externally managed` eNodeb devices. 

```suggestion
3. Magma **managed** eNBs refer to the radio devices that use TR-069 server communication using the enodebd service. **Externally managed** eNBs, in contrast, do not use TR-069 and are expected to be already configured to match the Cell ID and other configuration parameters so they successfully get connected to the AGW.
```